configure.py

```diff
@@ -305,7 +305,16 @@ def binary(name):
 else:
     n.variable('ar', configure_env.get('AR', 'ar'))
 
+def SearchPath(exe_name):
+  """Find an executable (.exe, .bat, whatever) in the system path."""
```

Comment:
Good point about the name and the comment. The function is OS-generic so I renamed it to search_system_path - it could be used on any OS, it just currently isn't. Thoughts?

Reply:
Thanks for the name change, I would just mention that the function is not OS-generic though. I.e. on Unix you would want to use ':', not ';' for the split, and you would need to handle empty items, which translate to "search the current path" (and frequent source of conflict and security issues, happens all the time with LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/some/new/path, when LD_LIBRARY_PATH was empty, and now your library search path will always start in the current directory, boom).

Please do not handle this here, but that's why `search_windows_path()` was a simpler name :-) And path searching is always trickier than expected.

Reply:
Good points. I could rename to search_windows_path, or leave it as-is. Fixing the path separator is easy enough (os.pathsep instead of ';', perhaps a worthwhile change anyway) but it sounds like it still wouldn't be _truly_ generic.